lib/core/src/Cardano/Wallet/TokenMetadata.hs

```diff
@@ -198,8 +221,8 @@ data SubjectProperties = SubjectProperties
     -- Name and description are required, both others may be missing the
     -- response.
     , properties ::
-        ( Property "name"
-        , Property "description"
+        ( Maybe (Property "name")
+        , Maybe (Property "description")
```

Comment:
:thinking: Why `Maybe` here? These are two required properties.

Reply:
This type is just for the response from the metadata-server, where possibly any field may be missing.
The function `metadataFromProperties :: SubjectProperties -> Maybe AssetMetadata` handles the case of missing fields required by cardano-wallet.